<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundOptionsDialog</class>
 <widget class="QDialog" name="SoundOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sound Options</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#ReplyFileOfferDialog
{   
color: transparent;
background-color: transparent;
border: 1px solid transparent;
border-radius: 8px; 
}

#StyledDlgTitleBox
{   
color: black;
background-color: rgb(50,238,17);
border: 2px solid green;
border-top-left-radius: 8px; 
border-top-right-radius: 8px;
}

#StyledDlgTitleLabel
{   
color: black;
}

#StyledDlgItemsBox
{   
background-color: rgb(50,238,17);
border: 2px solid green;
border-top-color: green; 
border-bottom-left-radius: 8px; 
border-bottom-right-radius: 8px;
}
QPushButton#CancelButton
{
color: white;
font: bold 14px;
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_normal.png) 10;
padding: -10;
}
QPushButton#CancelButton:pressed
{
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_pressed.png) 10;
padding: -10;
}
QPushButton#AcceptButton
{
color: white;
font: bold 14px;
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_normal.png) 10;
padding: -10;
}
QPushButton#AcceptButton:pressed
{
border-width: 10;
border-image: url(:/MyP2PWeb/Resources/button_green_pressed.png) 10;
padding: -10;
}
QLabel#PermissionLabel
{
font: bold 14px;
}
QLineEdit#FileNameEdit
{
background-color: rgb(238,238,238);
}
QPlainTextEdit#OfferMsgEdit
{
background-color: rgb(238,238,238);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="m_TitleBarWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>130</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="StyledDlgItemsBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="m_StatusLabel">
        <property name="text">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="MutePhoneRingCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Mute Phone Ring</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="MuteNotifySouncCheckBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Mute Notify Sound</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>302</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="CmdButtonBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QPushButton" name="AcceptButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CancelButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header>TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
